pkgs/development/libraries/alglib/default.nix

```
@@ -0,0 +1,36 @@
+{ lib, stdenv, fetchurl, cmake, clang }:
+
+stdenv.mkDerivation rec {
```

You might be able to get away without using `rec`
```suggestion
stdenv.mkDerivation {
```
(`rec` makes it so you can access one element from this attrset in another element of the same one, but there's a performance hit FYI. Probably not that big of a deal.)

---

Without `rec` I get the error `error: undefined variable 'version' at /home/micas/Desktop/nixpkgs/pkgs/development/libraries/alglib/default.nix:8:57`